src/index.html

```diff
@@ -11,6 +11,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      class="form-big">
```

Comment:
```suggestion
    <form
      action="#"
      method="post"
      class="form-big"
    >
```
Для чого потрібні два класи form-big та form-small, якщо в них однакові стилі?